www/_base.njk

```diff
@@ -7,13 +7,23 @@
     <link href="https://fonts.googleapis.com/css?family=Montserrat|Open+Sans" rel="stylesheet">
   </head>
   <body class="{{ current.source }}">
+    <div id="fb-root"></div>
+    <script>
+      (function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));
+    </script>
```

Comment:
Any reason this script is at the top of the body tag instead of the bottom? Typically it's better to place at the bottom for improved performance. Source: [Browser Diet](https://browserdiet.com/#css-on-top-js-on-bottom)